h2o-py/h2o/backend/cluster.py

```diff
@@ -290,16 +223,48 @@ def timezone(self, tz):
         assert_is_type(tz, str)
         h2o.rapids('(setTimeZone "%s")' % tz)
 
-
     def list_timezones(self):
         """Return the list of all known timezones."""
         from h2o.expr import ExprNode
         return h2o.H2OFrame._expr(expr=ExprNode("listTimeZones"))._frame()
-
-
-    #-------------------------------------------------------------------------------------------------------------------
+    
+    def check_version(self, strict=False):
```

Comment:
moved from `h2o` module: sits more comfortably here

Reply:
Why do you think `check_version` is better name than `version_check`? I am just curious. :)

Reply:
For functions that do something, I think we should always prefer verbs (like `list_timezones` right above) and keep nouns to represent states or abstractions.
Java developers especially tends to abuse nouns: for an old classical blog post about the difficult (and I agree sometimes painful) relation between Java and nouns, I recommend https://steve-yegge.blogspot.com/2006/03/execution-in-kingdom-of-nouns.html :)

Reply:
So in this case, we should rename also the `_strict_version_check()` function. ;) I am just kidding, I understand, what you want to say.

Reply:
you know what? I was thinking exactly the same when I noticed it a few minutes later.and I also remember having difficulties to find a name for that (private) function: it surely shows that my Java background is still there in a corner of my brain :)
But to my defense:
- it's a private method, and I care less than for public ones.
- it's actually not doing anything really: the idea is to return the "real" values of the `strict_version_check` parameter after considering the config and/or environment settings.
Maybe it should be an adverb :)